### PR TITLE
adding common library logging config functionality

### PR DESCRIPTION
**Pull request by andrewmil**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 aiohttp==2.0.5
 gunicorn==19.7.1
+sdx-common>=0.4.0
```
Comment (JamesGardiner) [inline requirements.txt]: 0.5.1

**Comment by andrewmil** [comment]
looking into travis build fail now

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,9 @@ language: python
 python:
     - "3.5"
     - "3.4"
+before_install:
+    - git clone https://github.com/ONSdigital/sdx-common.git
+    - pip3 install ./sdx-common
 install:
     - make
```
Comment (JamesGardiner) [inline .travis.yml]: `make build`

**Review by JamesGardiner — CHANGES_REQUESTED** [review]

**Comment by JamesGardiner** [comment]
@andrewmil see my comments for what should be the fix

File: .travis.yml (hunk 2 above)
Comment (andrewmil) [reply .travis.yml]: Just spotted this as well, I pretty much copied the collect changes and didnt realise this was running slightly differently, next commit should have things sorted

**Review by JamesGardiner — CHANGES_REQUESTED** [review]
Can you update the README to reference the `make dev` target?

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**